xml/System.Net.Security/AuthenticatedStream.xml

```diff
@@ -182,9 +182,20 @@ The following example demonstrates displaying the properties of an authenticated
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Asynchronously releases the unmanaged and managed resources used by the <see cref="T:System.Net.Security.AuthenticatedStream" />.</summary>
+        <returns>A task that represents the asynchronous dispose operation.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This method enables you to perform a resource-intensive dispose operation without blocking the main thread. 
```

Comment:
It's not too ambiguous here but "main" thread is a bit of a misnomer. Maybe just "current" thread?

Reply:
Your comment makes sense to me; just consider that I took this paragraph from base class [`System.IO.Stream` documentation](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.disposeasync?view=netcore-3.0), perhaps we should modify that one as well.

Reply:
I wonder if we could reword the "enables you":

```suggestion
 This method allows performing resource-intensive dispose operations without blocking the main thread.
```

Regarding the main vs current, I think we should try to keep it in sync with IO.Stream, as much as possible.